Controller/WebUIController.php

```
@@ -166,13 +166,15 @@ public function createAction(Request $request, string $configName, string $local
             $message = $message->withLocale($locale);
             $this->validateMessage($message, ['Create']);
         } catch (MessageValidationException $e) {
-            return new Response($e->getMessage(), 400);
+            return new Response($e->getMessage(), Response::HTTP_BAD_REQUEST);
         }
 
         try {
             $storage->create($message);
         } catch (StorageException $e) {
             throw new BadRequestHttpException(\sprintf('Key "%s" does already exist for "%s" on domain "%s".', $message->getKey(), $locale, $domain), $e);
+        } catch (\Exception $e) {
```

Is it really the behavior we're looking for? I'm afraid hiding all exceptions like this will make debugging much harder in case we have a problem in a storage isn't it?

---

(this comment also applies for all others storage method calls (update & delete).

---

Without the `catch` statement you see only `Unknown error.` in the UI. IMO this hides the actual problem even more.